frontend/app/views/info/supporterInternational.scala.html

```diff
@@ -0,0 +1,72 @@
+@import com.gu.i18n.CountryGroup
+@import com.gu.memsub.Current
+@import com.gu.salesforce.Tier.Supporter
+@import com.gu.membership.PaidMembershipPlans
+@import views.support.PageInfo
+@(supporterPlans: PaidMembershipPlans[Current, Supporter],
+    pageInfo: PageInfo,
+    pageImages: Seq[model.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
```

Comment:
Yes, fair point, but I'm not sure whether we might start supporting more countries and sharing templates between them and it seemed like this might make refactoring a tiny bit easier. Plus we already had this implicit parameter in existing controllers